modules/noc/class_noc.php

```diff
@@ -24,6 +24,7 @@ class noc
 {
     public function MACtoIP($SearchMAC, $IP, $ReadCommunity)
     {
+        $IP = null;
```

Comment:
It's back again ;)
Will remove it myself, together with the other instance new with this PR